cmd/issuer-rest/startcmd/start.go

```diff
@@ -826,3 +846,19 @@ func getOauth2Config(authURL, tokenURL, redirectURL, clientID, secret string) *o
 
 	return config
 }
+
+func readProfiles(path string) ([]*operation.Profile, error) {
+	f, err := os.Open(path) //nolint:gosec
```

Comment:
It's a good idea to include the reason for a `nolint` (usually the linter actually enforces this, not sure why it's not here)

example: `//nolint:gosec // user-defined path required`